spec/support/rails_template.rb

```diff
@@ -7,6 +7,7 @@
 copy_file File.expand_path('../templates/cucumber.rb', __FILE__),                "config/environments/cucumber.rb"
 copy_file File.expand_path('../templates/cucumber_with_reloading.rb', __FILE__), "config/environments/cucumber_with_reloading.rb"
 
+gsub_file 'config/database.yml', /^  adapter:.*\n/, %[  adapter: "jdbcsqlite3"\n] if RUBY_PLATFORM == "java"
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.